src/utils/editor.ts

```diff
@@ -68,12 +76,17 @@ export const COMPONENTS: ComponentType[] = [
   'ListMeta',
   'InputGroupMeta',
   'BreadcrumbMeta',
+  'TabsMeta',
 ]
 
 export const AccordionWhitelist: ComponentType[] = COMPONENTS.filter(
   name => !ALERT_COMPONENTS.includes(name),
 )
 
+export const TabsWhiteList: ComponentType[] = COMPONENTS.filter(
```

Comment:
You should add following types in the rootComponents.filter to avoid crash :)

- TabList
- TabPanels
- Tabs
- TabPanel